src/Stratis.Bitcoin.Features.Api/ApiSettings.cs

```diff
@@ -153,9 +151,9 @@ public static void BuildDefaultConfigurationFile(StringBuilder builder, Network
             builder.AppendLine($"#Keep Alive interval (set in seconds). Default: 0 (no keep alive).");
             builder.AppendLine($"#keepalive=0");
             builder.AppendLine($"#Use HTTPS protocol on the API. Default is { false }.");
-            builder.AppendLine($"#usehttps={DefaultCertificateFileName}");
+            builder.AppendLine($"#usehttps={ false }");
             builder.AppendLine($"#Subject name of the certificate to use for https traffic encryption.");
-            builder.AppendLine($"#certificatefilename={DefaultCertificateFileName}");
+            builder.AppendLine($"#certificatefilepath=");
```

Comment:
Here we should totally describe what kind of Certificate do we expect. For example, which formats do we support – i.e. which extensions. We should also say that the certification should not be password protected

Reply:
added a ref to the appropriate MSDN doc, and mentioned password are not supported, that should do it hopefully.